lib/email/receiver.rb

```diff
@@ -286,7 +286,7 @@ def handle_bounce
         Email::Receiver.update_bounce_score(@from_email, SiteSetting.hard_bounce_score)
       end
 
-      if SiteSetting.enable_whispers? && @from_user&.staged?
+      if SiteSetting.whispers_allowed_groups.present? && @from_user&.staged?
```

Comment:
Is the `system` user always able to create a whisper, not matter which groups are allowed to whisper? 🤔

Reply:
Now that we restored the bypass for staff, we should fix this check?

Reply:
^^ `s/staff/admin/` but the question still remains 🤔

Reply:
`system` is always allowed to create a whisper because we have a bypass for all admins so the old behavior is kept.